### PR TITLE
Add amplify

### PR DESCRIPTION
**Pull request by mark0428**
AMPlify is an attentive deep learning model for antimicrobial peptide prediction.
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by mark0428** [comment]
@BiocondaBot please add label

File: recipes/amplify/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin/
+cp -r src/. $PREFIX/bin/
```
Comment (johanneskoester) [inline recipes/amplify/build.sh:4]: The bin folder should only contain the actual executables. I suggest to copy the contents of src into `$PREFIX/share/amplify`, and put a script like

```
#!/bin/bash

$PREFIX/share/amplify/amplify.py $@
```

into the bin folder. You will need to generate this via echo, such that $PREFIX ends up to be the correct path in the script.

**Review by johanneskoester — CHANGES_REQUESTED** [review]

File: recipes/amplify/build.sh (hunk 1 above)
Comment (mark0428) [reply recipes/amplify/build.sh]: Thanks for the suggestion! We have made changes to the scripts.

**Comment by mark0428** [comment]
Hi @johanneskoester. We have made changes as requested. Could you please review it again? Thanks a lot!

**Comment by warrenlr** [comment]
Hello,

Would you be able to give us an ETA for your review+merging this PR? 

Thank you for your prompt attention into the matter,

Rene

**Comment by mark0428** [comment]
Hi @johanneskoester. It's been a while since we made the changes. Could you please take a look at this? Many thanks for your efforts!

**Comment by kmnip** [comment]
@bioconda/<team> Can you please approve and merge the PR? Thank you.

**Review by druvus — APPROVED** [review]

**Comment by druvus** [comment]
@BiocondaBot please merge

**Comment by simonbray** [comment]
@bioconda-bot please update

**Comment by simonbray** [comment]
@BiocondaBot please merge

**Comment by simonbray** [comment]
I suspect @johanneskoester needs to re-review before this can be merged.

**Comment by lcoombe** [comment]
Hi @simonbray, @druvus,

Is there anything more that can be done to get this PR merged? It's been almost a month since @johanneskoester requested changes, and we'd really like to get this recipe added to bioconda. 
If it's easier, @mark0428 can also close this and open a fresh PR.

Thanks for your help!
Lauren

**Comment by simonbray** [comment]
Hi @lcoombe, if you open a new PR, I'll merge it.

**Pull request closed without merging**